contracts/modules/cover/CoverProducts.sol

```diff
@@ -320,7 +321,16 @@ contract CoverProducts is ICoverProducts, MasterAwareV2, Multicall {
     for (uint i = 0; i < _productsToMigrate.length; i++) {
       _products.push(_productsToMigrate[i]);
       productNames[i] = _cover.productNames(i);
-      allowedPools[i] = _cover.allowedPools(i);
+
+      /*
+      * Currently only products with fixed price have allowed pools
+      * And all of them have only one allowed pool that's why we fetch only the first one
+      */
+      if (_productsToMigrate[i].useFixedPrice && !_productsToMigrate[i].isDeprecated) {
+        uint _allowedPool = _cover.allowedPools(i, 0);
+        allowedPools[i] = [_allowedPool];
```

Comment:
we have just added a product that has 3 allowed pools, instead of 1 like the others... id 186

Reply:
Yeah, I saw it. I will close this PR and create another one with a different approach.